modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/NA.json

```diff
@@ -0,0 +1,58 @@
+[
+	{
+		"name": "//Karas",
```

Comment:
Is it correct?

Reply:
Yes, I follow here https://www.iso.org/obp/ui/#iso:code:3166:NA